Gemfile.lock

```diff
@@ -436,14 +427,14 @@ GEM
       addressable (>= 2.3.6)
       crack (>= 0.3.2)
       hashdiff
-    webpacker (4.0.7)
+    webpacker (4.2.2)
       activesupport (>= 4.2)
       rack-proxy (>= 0.6.1)
       railties (>= 4.2)
     websocket-driver (0.7.1)
       websocket-extensions (>= 0.1.0)
     websocket-extensions (0.1.4)
-    zeitwerk (2.1.10)
+    zeitwerk (2.2.2)
```

Comment:
### [zeitwerk](https://github.com/fxn/zeitwerk)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.1.10 → 2.2.2

[_~~change-log~~, ~~source-code~~_]





<details>
<summary>Commits</summary>

A change of **39** commits. See the full changes on [the compare page](https://github.com/fxn/zeitwerk/compare/v2.1.10...v2.2.2).

These are the first 10 commits:
- (8c4aadb) [explains why setup before main module definition](https://github.com/fxn/zeitwerk/commit/8c4aadb822b59c2170e114ec06791dd426f2ecdf)
- (1b0d316) [adds a remark about eager loading and gems](https://github.com/fxn/zeitwerk/commit/1b0d3167535e1b4b60019b778a5baafec096fb8d)
- (585569c) [documents a way to extract custom inflectors for gems](https://github.com/fxn/zeitwerk/commit/585569cd8e345edc3676e32dda6b201b185e00b2)
- (8c4c7ba) [temporarily allow ruby-head to fail in CI](https://github.com/fxn/zeitwerk/commit/8c4c7ba7075ebd9e451ca2184b2bfc6b106f565c)
- (d3e56c6) [Route back to Trusty image](https://github.com/fxn/zeitwerk/commit/d3e56c6061620a78dde7dff4462873913dfdc08f)
- (20825c1) [Merge pull request #81 from cotsog/patch-1](https://github.com/fxn/zeitwerk/commit/20825c1f32896a68b0a3d6ac75dfb26c531f2617)
- (dadc74d) [fixes badge URL [skip ci]](https://github.com/fxn/zeitwerk/commit/dadc74d8684fc789e55cc53b2f2890c38792f55e)
- (b958f7e) [documents gotcha autoloading explicit namespaces in debugger…](https://github.com/fxn/zeitwerk/commit/b958f7e5f054d1ee3234097320788f5c44765676)
- (d0965e2) [revises link in README [skip ci]](https://github.com/fxn/zeitwerk/commit/d0965e287560489fea9e6bc7361e03377fcdddda)
- (3843af5) [correction [slip ci]](https://github.com/fxn/zeitwerk/commit/3843af5be157803f052ac0e3c9e5ae17763dc893)


</details>